google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -639,6 +639,9 @@ def rotate encryption_key: nil, new_encryption_key: nil
         # Permanently deletes the file.
         #
         # @return [Boolean] Returns `true` if the file was deleted.
+        # @param [Boolean,Integer] generation Select a specific revision of the
+        #   file to delete. When `true` it will use the object's generation. The
```

Comment:
I'm confused here. What is the value returned by `#generation`, if it is not the "latest version"? In other words, what is the purpose of passing `true` vs. the default. (Also, google-cloud-storage uses "file" rather than "object".)